checkov/common/bridgecrew/platform_integration.py

```diff
@@ -55,7 +55,7 @@
     normalize_bc_url,
     REQUEST_CONNECT_TIMEOUT,
     REQUEST_READ_TIMEOUT,
-    REQUEST_RETRIES,
+    REQUEST_RETRIES, REQUEST_STATUS_CODES_RETRY, REQUEST_METHODS_TO_RETRY,
```

Comment:
are you using these imports? I see youv'e defined them as consts below

Reply:
Right, I'll delete them. 
Thank you!